### PR TITLE
Feature: comscore analytics connector

### PR DESCRIPTION
**Pull request by OlegRyz**
Add Comscore analytics connector from React Native connector's repository

### CODE REVIEW

**Comment by hovig-theo** [comment]
All the newly added Comscore classes are defining the package as `package com.theoplayercomscore.integration`, meanwhile the actual package should be `package com.theoplayer.android.connector.analytics.comscore`.

File: connectors/analytics/comscore/build.gradle [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    kotlinOptions {
+        jvmTarget = '1.8'
+    }
+}
+
+dependencies {
+    compileOnly "com.theoplayer.theoplayer-sdk-android:core:$sdkVersion"
+    compileOnly "com.theoplayer.theoplayer-sdk-android:integration-ads-ima:$sdkVersion"
+
+    implementation "com.comscore:android-analytics:6.+"
```
Comment (hovig-theo) [inline connectors/analytics/comscore/build.gradle:37]: Normally we add the third party SDK as a `compileOnly` and then use the `implementation` in the `app/build.gradle`.

**Review by hovig-theo — CHANGES_REQUESTED** [review]

**Review by tvanlaerhoven — APPROVED** [review]

File: app/src/main/java/com/theoplayer/android/connector/MainActivity.kt [hunk 2]
```diff
@@ -30,9 +35,65 @@ class MainActivity : AppCompatActivity() {
...
+            mediaFormat = null,
+            distributionModel = null,
+            playlistTitle = null,
+            totalSegments = null,
+            clipUrl = null,
+            videoDimension = null,
+            customLabels = emptyMap(),
+        )
+        comscoreConnector =
+            ComscoreConnector(this, theoplayerView.player, comscoreConfiguration, metadata)
```
[the remark this answers is not in the document]
Comment (OlegRyz) [reply app/src/main/java/com/theoplayer/android/connector/MainActivity.kt]: Done

File: connectors/analytics/comscore/build.gradle (hunk 1 above)
Comment (OlegRyz) [reply connectors/analytics/comscore/build.gradle]: Done

**Comment by OlegRyz** [comment]
> All the newly added Comscore classes are defining the package as `package com.theoplayercomscore.integration`, meanwhile the actual package should be `package com.theoplayer.android.connector.analytics.comscore`.

Indeed, good catch!

File: connectors/analytics/comscore/build.gradle (hunk 1 above)
Comment (Danesz) [reply connectors/analytics/comscore/build.gradle]: how will this be published to Maven? Who will bring in the dependency? Should we, or do we document it for the customers?

Comment (OlegRyz) [reply connectors/analytics/comscore/build.gradle]: There is publish.gradle script that does publishing. It picks up projects and publishes them to maven. @hovig-theo knows more about that, I guess

Comment (hovig-theo) [reply connectors/analytics/comscore/build.gradle]: Indeed the `publish.gradle` file prepares the maven publishing which is picked up by the Jitpack.

Regarding the dependencies, the user will have to add it and that gives them the benefit to change the version in their app as long as there's no breaking change from the third-party SDK.

**Review by hovig-theo — APPROVED** [review]

**Pull request merged**